_schemaV2.graphql

```diff
@@ -13423,6 +13428,11 @@ type Mutation {
   # Update a collection
   updateCollection(input: updateCollectionInput!): updateCollectionPayload
 
+  # Updates the user's collections in batch.
+  updateCollectionsMutation(
```

Comment:
Naming suggestion: Given the mutation names are global in the schema, without name spacing, let's prepare ourselves for 2 different mutations:

- update collections for myself (based on the access token): our needs
- update collections for a user (e.g. performed by an admin, with a user id param): future use cases

I can't find a consistent pattern in MP for mutations, but I'd suggest we use name

- `updateMeCollectionsMutation`: update collections for myself (I kind of like `updateMyCollectionsMutation` but apparently "MyCollection" means something else already 😞 .)
- `updateUserCollectionsMutation`: update collections for a user

Reply:
And we can also update the input, payload, response types accordingly.

Reply:
👍